Motorcycle.js

```diff
@@ -1,6 +1,33 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle(make, model, year, color, seats, passengers) {
+	if (this.seats <= 2) {
+	Car.call(this, make, model, year, color, seats, passengers);
+	} else {
+		Car.call(this, make, model, year, color, 2, passengers);
```

Comment:
This is an interesting approach! I think a more standard way to do it would be the following:
```
function Motorcycle(make, model, year, color, seats, passengers) {
	Car.call(this, make, model, year, color, seats, passengers);
	if (this.seats > 2) {
		this.seats = 2;
	}
};
```